ocrd/ocrd/constants.py

```diff
@@ -1,7 +1,7 @@
 """
 Constants for ocrd.
 """
-from ocrd_utils.package_resources import resource_filename
+from pkg_resources import resource_filename
```

Comment:
What is wrong with `ocrd_utils.package_resources.resource_filename`?

Reply:
Or rather, should we adopt `ocrd_utils.package_resources` to provide the basic `pkg_resources` functionality, so it is consistent?

Reply:
OK, reading commit-by-commit would be wise, `ocrd_utils.package_resources` is gone now, which is also a solution.

Reply:
Like I said above, I have simply reverted the merge of #882 here.